spec/02-integration/07-sdk/06-worker_events_spec.lua

```diff
@@ -0,0 +1,105 @@
+local helpers = require "spec.helpers"
+
+describe("worker_events", function()
+  local strategy = "off"
+  local business_port
+
+  lazy_setup(function()
+    business_port = helpers.get_available_port()
+    local fixtures = {
+      http_mock = {
+        worker_events = [[
+          server {
+            server_name example.com;
+            listen %s;
+
+            location = /test {
```

Comment:
Could we split this location into two locations? one for string, one for table, and use two `it()` to test them.

Reply:
done